src/Sarif.Viewer.VisualStudio.Core/XamlUtilities.cs

```diff
@@ -0,0 +1,113 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Markup;
+using System.Windows.Navigation;
+
+using EnvDTE;
+
+using EnvDTE80;
+
+using Microsoft.Sarif.Viewer;
```

Comment:
Remove

Reply:
the namespace 'Sarif.Viewer.VisualStudio.Core' was added automatically when create a new file. Updated it to 'Microsoft.Sarif.Viewer' as all other files in the project. Fix all references as well.